README.md

```diff
@@ -1,2 +1,75 @@
-# client-maven-plugin
-Plugin that simplifies using Gluon Client for Java/JavaFX maven projects
+# The Gluon Client plugin for Maven
+
+The Gluon Client plugin for maven projects leverages GraalVM, OpenJDK and JavaFX 11+, 
+by compiling into native code the Java Client application and all its required dependencies, 
+so it can directly be executed as a native application on the target platform.
+
+## Getting started
+
+To use the plugin, apply the following steps:
+
+### 1. Apply the plugin
+
+Edit your pom file and add the plugin:
+
+    <plugin>
+    
+        <groupId>com.gluonhq</groupId>
+        <artifactId>client-maven-plugin</artifactId>
+        <version>0.0.1</version>
+        <configuration>
+            <mainClass>your.mainClass</mainClass>
+        </configuration>
+    </plugin>
+
+The plugin allows some options that can be set in `configuration`, to modify the default settings, and several goals, to build and run the native application.
+
+### 2. Goals
+
+You can run the regular goals to build and run your project as a regular Java project, if you use the `javafx-maven-plugin` plugin:
```

Comment:
May be re-write this as:

`javafx-maven-plugin` is internally applied by the `client-maven-plugin` allowing one to build and run the project using:

    mvn clean javafx:run

Reply:
You actually have to combine both plugins. `client-maven-plugin` doesn't use `javafx-maven-plugin`, you still have to do `javafx:run` to run as usual on the VM, and `client:run` to run the native image.

Reply:
Ah, I see. My bad.

Reply:
I've tweaked it a little bit